### PR TITLE
MIC-291 Update layout of referral/contributory-factors-comments screen

### PR DESCRIPTION
**Pull request by bill-moj**
MIC-291

image

### CODE REVIEW

File: server/journeys/referral/contributory-factors-comments/tests.test.ts [hunk 1]
```diff
@@ -68,12 +68,13 @@ describe('GET /referral/contributory-factors-comments', () => {
...
     expect(getByText(html, 'Add comments about the contributory factors (optional)')).toBeVisible()
-    const addCommentLink = getByRole(html, 'link', { name: 'Add comment for TEXT factor' }) as HTMLLinkElement
+    const addCommentLink = getByRole(html, 'link', { name: 'Add comment on TEXT factor' }) as HTMLLinkElement
     expect(addCommentLink).toBeVisible()
     expect(addCommentLink.href.endsWith('a-comment')).toBeTruthy()
-    const editCommentLink = getByRole(html, 'link', { name: 'Edit comment for TEXT2 factor' }) as HTMLLinkElement
+    const editCommentLink = getByRole(html, 'link', { name: 'Edit comment on TEXT2 factor' }) as HTMLLinkElement
     expect(editCommentLink).toBeVisible()
     expect(editCommentLink.href.endsWith('b-comment')).toBeTruthy()
+    expect(queryByText(html, 'Sample Comment Text')).toBeVisible()
```
Comment (ReedSoftware) [inline server/journeys/referral/contributory-factors-comments/tests.test.ts:77]: Why not getByText?

Comment (bill-moj) [reply server/journeys/referral/contributory-factors-comments/tests.test.ts:77]: ahh carryover from what i did for CYA test. 
queryByText allows me to check for negative result, ie, `.not.toBeInTheDocument()`, this seems more flexible than getByText in its usage.

Comment (ReedSoftware) [reply server/journeys/referral/contributory-factors-comments/tests.test.ts:77]: I think getByText gives you better errors if it doesn't find it - better to use it in general unless you need `query` for the reasons you described like it not existing etc

Comment (bill-moj) [reply server/journeys/referral/contributory-factors-comments/tests.test.ts]: ya getByText error msg is more helpful when something go wrong with the test

**Review by ReedSoftware — APPROVED** [review]

**Pull request merged**